packages/client/src/pages/Login/Login.tsx

```diff
@@ -84,13 +87,23 @@ const Login = () => {
           </>
         </CustomForm>
 
-          <Link className={styles['login__link--help']} to={'/'}>
+          <Box className={styles['login__link--title']}>
             Войти с помощью
-          </Link>
+          </Box>
 
-          <div className={styles.login__logo}>
-            <Logo></Logo>
-          </div>
+          <Button
+            className={styles.login__logo}
+            variant="link"
+            onClick={async () => {
+              const { data, isError } =  await refetchServiceId()
```

Comment:
А здесь дернуть эндпоинт на прямую

```suggestion
              const { data, isError } =  await dispatch(oauthApi.endpoints.getServiceId.initiate(OAUTH_REDIRECT_URL))
```